src/containers/Transactions/transaction.scss

```diff
@@ -25,17 +25,17 @@
     margin: 8px auto 0px;
 
     @include for-size(tablet-landscape-up) {
```

Comment:
Why were these values changed?

Reply:
Anywhere that had a hardcoded width had to be adjusted to include the padding and borders.